src/main/resources/application.yml

```diff
@@ -1,3 +1,9 @@
 server:
   # TODO: Update server port to a unique value.
-  port: 8080
+  port: 8207
+spring:
+  data:
+    mongodb:
+      host: localhost
```

Comment:
Need to include env vars for these, see https://github.com/Health-Education-England/TIS-TRAINEE-DETAILS/blob/chore/buildPipeline/src/main/resources/application.properties

Reply:
done